.github/ISSUE_TEMPLATE/02-chaosscast-episode-checklist.md

```diff
@@ -0,0 +1,89 @@
+---
+name: CHAOSScast Episode Release Checklist
+about: Use this template to track the tasks required to release an episode of CHAOSScast
+labels: ['CHAOSScast']
+
+---
+
+Please get approval from the CHAOSScast facilitator before creating a CHAOSScast episode release checklist issue. You can learn more about the process for CHAOSScast by visiting our [CHAOSScast process documentation](https://github.com/chaoss/community/blob/main/media-and-outreach/chaosscast.md)
+
+## CHAOSScast Episode Release Checklist
+
+### Scheduling an episode
+
+* **Step 1:** Propose a topic and guest(s) \(Anyone\)
+  * [ ] **Action:** Discuss with facilitator to get approval for the episode topic and guest.
+  * [ ] **Action:** Recruit interview guest to finalize topic and content of the podcast together
+* **Step 2:** Facilitate Episode \(Facilitator\)
+  * [ ] **Action:** Send invitation email to guest to find a few potential suitable dates and times \(60-90 minutes.\)
+  * [ ] **Action:** Notify organizer of guest's date and time possibilities, and confirm one.
+  * [ ] **Action:** Prepare the planning document (copy the [Episode Planning Document template](https://docs.google.com/document/d/17qjAafr0aAWXV_HR-WiefHOa881t8X7tEoiZnPBh3ao/edit?pli=1#heading=h.cmpmu92ylw4d) and start filling in information.)
+  * [ ] **Action:** Invite panelists from the CHAOSS community who have expertise in the episode topic.
+  * [ ] **Action:** Confirm panelists 
+  * [ ] **Action:** Send calendar invitation with confirmed date of the podcast 
+    * The event has to be created in the [CHAOSScast calendar](https://calendar.google.com/calendar/ical/513r47bb756t07lp5tat5nsuj0%40group.calendar.google.com/public/basic.ics)
+      * All organizers should have full access to the calendar \(i.e., Make changes and manage sharing\)
+    * Name calendar event:🎙  CHAOSScast {Topic}
+      * Start with the microphone emoticon \(🎙 \) to signal recording meeting
+  * [ ] **Action:** Send out a [confirmation email](https://docs.google.com/document/d/1Pw8OKn-qSSkMw573BZu-EZ4BZualesdZjPbpBEKvzrc/edit) that includes the [“how we podcast” document](https://docs.google.com/document/d/1j5Ng91rrFrx3PdPejv26rBxb1P7UX-O_Q0_I9TCIya0/edit#) to guests and panelists
+  * [ ] **Action:** Update the episode planning document with recording time and zoom link
```

Comment:
This action can be removed. We have the calendar invite for the date and zoom info.